hedera-node/hedera-app/src/main/java/com/hedera/node/app/ServicesMain.java

```diff
@@ -242,9 +260,6 @@ public static void main(final String... args) throws Exception {
         final var merkleCryptography = MerkleCryptographyFactory.create(configuration, cryptography);
         MerkleCryptoFactory.set(merkleCryptography);
 
-        // Register the metrics related to TSS functionalities
-        hedera.registerTssMetrics(metrics);
```

Comment:
Some of these changes are merged in my PR also. You might have to merge back develop

Reply:
Done